src/libsync/syncengine.cpp

```diff
@@ -214,6 +214,9 @@ bool SyncEngine::checkErrorBlacklisting( SyncFileItem &item )
         } else if( item._modtime != entry._lastTryModtime ) {
             qDebug() << item._file << " is blacklisted, but has changed mtime!";
             return false;
+        } else if( item._renameTarget != entry._renameTarget) {
+            qDebug() << item._file << " is blacklisted, byt renameTarget changed from" << entry._renameTarget;
```

Comment:
that's what every other branch in this function does.  I don't think one need a comment for that in this branch and not the others.